src/components/scenes/ChangeMiningFeeScene.tsx

```diff
@@ -186,4 +213,16 @@ const getStyles = cacheStyles((theme: Theme) => {
   }
 })
 
-export const ChangeMiningFeeScene = withTheme(ChangeMiningFeeComponent)
+const ChangeMiningFeeSceneThemed = withTheme(ChangeMiningFeeComponent)
+
+export const ChangeMiningFeeScene = (props: OwnProps) => {
+  const account = useSelector(state => state.core.account)
+  const currencyCode = useSelector(state => state.ui.wallets.selectedCurrencyCode)
+  const walletId = useSelector(state => state.ui.wallets.selectedWalletId)
+  const wallet = account.currencyWallets[walletId] ?? {}
+  const { pluginId = '' } = wallet.currencyInfo ?? {}
+  const tokenId = getTokenId(account, pluginId, currencyCode)
+
+  const iconColor = useIconColor({ pluginId, tokenId: tokenId !== undefined ? tokenId : null })
```

Comment:
don't default tokenId to null as that will pull the parent asset icon color

Reply:
Isn’t that the goal? If there is no token id then this would mean the wallet is not a token wallet but a mainnet wallet.